plottr/plot/mpl.py

```diff
@@ -656,9 +656,20 @@ def __init__(self, name: str, parent: Optional[QtWidgets.QWidget] = None):
         # other options
         self.addSeparator()
 
-        self.plotComplexPolar = self.addAction('Mag/Phase')
-        self.plotComplexPolar.setCheckable(True)
-        self.plotComplexPolar.triggered.connect(self._trigger_complex_mag_phase)
+        self.plotComplexReIm = self.addAction('Real/Imag')
+        self.plotComplexReIm.setCheckable(True)
+        self.plotComplexReIm.triggered.connect(
+            lambda: self.selectComplexType(ComplexRepresentation.realAndImag))
+
+        self.plotReal = self.addAction('Real')
+        self.plotReal.setCheckable(True)
+        self.plotReal.triggered.connect(
+            lambda: self.selectComplexType(ComplexRepresentation.real))
+
+        self.plotMagPhase = self.addAction('Mag/Phase')
+        self.plotMagPhase.setCheckable(True)
+        self.plotMagPhase.triggered.connect(
+            lambda: self.selectComplexType(ComplexRepresentation.magAndPhase))
```

Comment:
I think it would be more logic if the order is. Real, Real/Imag, Mag/Phase rather than having real in the middle